docker/alpine-base.dockerfile

```diff
@@ -6,3 +6,8 @@ WORKDIR /app
 RUN apk add --no-cache iputils setpriv dumb-init python3 py3-cryptography py3-pip py3-six py3-yaml py3-click py3-markdown py3-requests py3-requests-oauthlib && \
     pip3 --no-cache-dir install apprise==0.9.8.3 && \
     rm -rf /root/.cache
+
+# Install grpcurl
+# Can be used like ./grpcurl
+RUN wget https://github.com/fullstorydev/grpcurl/releases/download/v1.8.6/grpcurl_1.8.6_linux_x86_64.tar.gz
+RUN tar xf grpcurl_1.8.6_linux_x86_64.tar.gz
```

Comment:
Can you create an argument variable to have version number in one place, instead of 3 places?

Reply:
Done via adding ARG